docs/source/usage/exporting_metaframes.rst

```diff
@@ -0,0 +1,308 @@
+* Change variable types
+* Modify distribution parameters
+* Adjust missing data rates
+
+Let's say we use the exported MetaFrame as provided above to synthesize 5 rows of data, resulting in the following dataset:
+
+.. list-table::
+   :widths: 10 20 10 20 20
+   :header-rows: 1
+
+   * - ID (i64)
+     - fruits (cat)
+     - B (i64)
+     - cars (cat)
+     - optional (i64)
+   * - 1
+     - apple
+     - 1
+     - beetle
+     - 287
+   * - 2
+     - banana
+     - 2
+     - beetle
+     - 265
+   * - 3
+     - apple
+     - 6
+     - beetle
+     - 152
+   * - 4
+     - banana
+     - 0
+     - beetle
+     - null
+   * - 5
+     - banana
+     - 5
+     - audi
+     - 87
+
+Well, what if we wanted to change the distribution of the ``fruits`` variable to instead be 30% ``apple``, 30% ``banana``, and introduce a new fruit ``orange`` with a distribution of 40%? We can do this by editing the ``probs`` and ``labels`` attributes of the ``fruits`` variable in the exported MetaFrame. The following is the edited MetaFrame:
+
+
+.. tab:: JSON file before
+
+    .. code-block:: json
+
+        // ...
+        {
+                "name": "fruits",
+                "type": "categorical",
+                "dtype": "Categorical",
+                "prop_missing": 0.0,
+                "distribution": {
+                    "implements": "core.multinoulli",
+                    "provenance": "builtin",
+                    "class_name": "MultinoulliDistribution",
+                    "parameters": {
+                        "labels": [
+                            "apple",
+                            "banana"
+                        ],
+                        "probs": [
+                            0.4,
+                            0.6
+                        ]
+                    }
+                }
+            },
+            // ...
+
+.. tab:: JSON file after
+    
+    .. code-block:: json
+        :emphasize-lines: 15, 18, 19, 20
+
+        // ...
+        {
+                "name": "fruits",
+                "type": "categorical",
+                "dtype": "Categorical",
+                "prop_missing": 0.0,
+                "distribution": {
+                    "implements": "core.multinoulli",
+                    "provenance": "builtin",
+                    "class_name": "MultinoulliDistribution",
+                    "parameters": {
+                        "labels": [
+                            "apple",
+                            "banana",
+                            "orange"
+                        ],
+                        "probs": [
+                            0.3,
+                            0.3,
+                            0.4
+                        ]
+                    }
+                }
+            },
+            // ...
+
+
+If we now synthesize five rows of data using the edited MetaFrame, we get the following dataset, which as you can see contains the added ``orange`` fruit, and follows the new distribution:
```

Comment:
Yeah you're right, in general I've referred to the exported metaframe as a metaframe that's simply been serialized and exported to json, which is why I've ended up with this phrasing. That said I realize this specific sentence implies that you're not during the object directly instead of the gmf, so I'll change it to be more reflective of that!

Reply:
Also, I like your analogy. I looked at the docs, and think it would make sense to refer to the exported objects as GMF files as opposed to either 'exported metaframes/json files/etc.'. Only thing that I struggle with is that 1) json files are 'recognisable' as such, they might confuse the reader less, and 2) the function to export them is 'to_json', should the function maybe be renamed to something more easily understandable (such as 'export' or 'to_gmf')?

Let's discuss the usage of GMF/JSON with in person later today!